regex-assembly/932239.ra

```diff
@@ -11,5 +11,5 @@
 ##! These patterns are approximations of the patterns used by the cmdline
 ##! processor for `@` and `~`.
 ##! These patterns are used across multiple files, change with care.
-##!> include-except unix-shell-upto3 unix-shell-fps-pl2 unix-shell-fps-pl2-start-of-string unix-shell-fps-useragents -- @ [\s<>&|)] ~ \S
-##!> include-except unix-shell-4andup unix-shell-fps-pl2 unix-shell-fps-pl2-start-of-string unix-shell-fps-useragents -- @ [\s<>&|)] ~ \S
+##!> include-except unix-shell-upto3 unix-shell-fps-pl2 unix-shell-fps-pl2-start-of-string unix-shell-fps-useragents -- @ (?:[\s<>&|),]|$) ~ \S
+##!> include-except unix-shell-4andup unix-shell-fps-pl2 unix-shell-fps-pl2-start-of-string unix-shell-fps-useragents -- @ (?:[\s<>&|),]|$) ~ \S
```

Comment:
I'd rather adding `,` be done in another PR, this PR will likely include some regressions and I don't want to introduce more false positives with this PR.

Reply:
If you create a follow-up PR right after I'm ok with it. This expressions should be consistent, so it must be consistent across all files for the next release.

Reply:
ok nevermind, I see it came from `toolchain.yaml`. I can test these new changes as well if you want to be sure it doesn't cause any new false positives.